CHANGELOG.md

```diff
@@ -1 +1,14 @@
 # Changelog
+
+## [0.0.3] - 2019-06-02
+
+### Added 
+* Added variables to define strings used in `_volcano.py` graph labels.  
```

Comment:
@shammamah  @mkcor : we could also add "Add tests framework for dash-bio components and demo app testing", no?

Reply:
I don't think that has anything to do with the actual `dash-bio` package, so I left it out.